recipes/xorg/all/conanfile.py

```diff
@@ -73,6 +73,15 @@ def system_requirements(self):
                            "libXScrnSaver", "xcb-util-wm", "xcb-util-image", "xcb-util-keysyms", "xcb-util-renderutil",
                            "libxxf86vm", "libxv", "xkeyboard-config", "xcb-util", "xcb-util-cursor"], update=True, check=True)
 
+        if conan_version > "2.0":
```

Comment:
```suggestion
        if Version(conan_version) >= "2.0.10":
```
APK is supported since 2.0.10: https://docs.conan.io/2/changelog.html#aug-2023

I would convert to Version just in case, because it's not documented in docs.conan.io